packages/multichain/jest.config.js

```diff
@@ -17,10 +17,10 @@ module.exports = merge(baseConfig, {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 100,
+      branches: 97.55,
       functions: 100,
-      lines: 100,
-      statements: 100,
+      lines: 99.83,
+      statements: 99.84,
```

Comment:
It would be great if we could keep this at 100. I have made some suggestions that should get us closer.

``` suggestion
      branches: 100,
      functions: 100,
      lines: 100,
      statements: 100,
```

Reply:
Coverage is now back up to 100.